### DISCUSSION TITLE
Feature request: Adjusting volume via scroll wheel while hovering the notification tray icon.

### DISCUSSION THREAD

**Discussion by cgohier** [ShankarBUS/ModernFlyouts#881] [Ideas]
The flyout would (optionally maybe) open while adjusting the volume with the scroll wheel over the tray icon.

This functionality exists in earTrumpet and various Linux desktop environments.

**Comment by prakharb5** [comment]
Can you please open an issue for this? Discussions are not the correct way to open feature requests :)
